tests/zfs-tests/tests/functional/cli_root/zpool_expand/zpool_expand_003_neg.ksh

```diff
@@ -27,23 +27,23 @@
 
```

Comment:
This is similar to `zpool_expand_001_pos` test but is testing that opt-out of auto-expand prevents expanding the pool.  The auto-expand automation logic in ZED uses the `devid` and `phys_path` keys for vdev matching.  These keys are not present for file based vdevs so the changes below will always report success and doesn't really test autoexpand opt-out for disks.

I would recommend excluding test 003 until we have test 001 working.  The auto-expand code exists in ZED but we propbably need to use DM disks during the test for the matching to succeed.